de/manual/ch_Tools_Assistants.xml

```diff
@@ -73,33 +73,48 @@
-        werden nur die zu diesem Konto zugehörigen Buchungsteile durchsucht, es sei denn, Sie
-        wählen als Suchkriterium <guilabel>Alle Konten</guilabel>.
-        <footnote id="fn_tools_all_accounts">
-          <para>Die Auswahl <guilabel>Alle Konten</guilabel> führt eine Suche durch, bei der Buchungen in den
-            Konten, die im Dialog <guilabel>Konten wählen</guilabel> markiert wurden, gefunden
-            werden, die mindestens einen Buchungsteil in jedem der ausgewählten Konten enthalten.
-            Wenn die Suche in einem regulären Kontobuch gestartet wird, ist dies das einzige
-            Kriterium, das Buchungsteile von anderen Konten prüft. Wenn Sie also den Dialog
-            <guilabel>Buchungen suchen</guilabel> z.B. in Aktiva:Umlaufvermögen:Girokonto öffnen
-            und alle Buchungen dieses Kontos mit einem Buchungsteil in Ausgaben:Lebensmittel finden
-            wollen, ist dies als Kriterium zu verwenden. Beachten Sie jedoch, dass es sich um eine
-            <emphasis>UND</emphasis>-Suche handelt: Wenn Sie zusätzlich auch nach
-            Ausgaben:Verschiedenes suchen, erhalten Sie <emphasis>nur</emphasis> die Buchungen mit
-            einem Buchungsteil in jedem der beiden Konten und <emphasis>nicht</emphasis> die
-            Buchungen mit nur einem einzigen Buchungsteil.
-          </para>
-        </footnote>
-        Möchten Sie stattdessen das gesamte &app;-Buch durchsuchen, dann starten Sie die Suche
-        entweder in der <emphasis>Kontenübersicht</emphasis>, oder Sie öffnen das
-        &mc.to.gen-jour;.
-      </para>
+      <para>Abhängig davon, wie tief Sie in Ihren Daten abtauchen wollen, können die folgenden
+        <emphasis>Ausgangssituationen</emphasis> unterschieden werden:
+        <variablelist>
+          <varlistentry>
+            <term>Ein einzelnes <emphasis>Konto</emphasis> durchsuchen</term>
 
-      <para>Bei den Buchungsfeldern werden nur die Buchungen berücksichtigt, die mit einem Buchungsteil in
-        diesem Register verknüpft sind, während bei den übrigen Buchungen nur die mit dem Konto
-        verbundenen Buchungsteile durchsucht werden, die diesem Konto zugeordnet sind, außer Sie
-        verwenden die Suchfunktion <guilabel>Alle Konten</guilabel>.
+            <listitem>
+              <para>Führen Sie die Suche in der Registerkarte eines Kontos durch, so werden nur die zu diesem Konto
+                zugehörigen Buchungsteile durchsucht. Möchten Sie doch eine Suche über mehrer
+                Konten durchführen, verwenden Sie als Suchkriterium <guilabel>Alle
+                Konten</guilabel>.
+                <footnote id="fn_tools_all_accounts">
+                  <para>Das Suchkriterium <guilabel>Alle Konten</guilabel> führt eine Suche durch, bei der Buchungen in den
+                    Konten, die im dem Dialog, nach betätigen der Schaltfläche <guilabel>Konten
+                    wählen</guilabel>, markiert wurden, gefunden werden und mindestens einen
```

Comment:
Ich verstehe imme noch nicht, wo `Das Suchkriterium <guilabel>Alle Konten</guilabel>` oder ` Schaltfläche <guilabel>Konten wählen</guilabel>` sein soll.

Reply:
mhh, ok vielleicht ist diese kleine Auflistung mit der Fußnote an dieser Stelle im Text noch zu früh platziert.

Im Bild des Dialogs sieht man gut, dass das Suchkriterium in den einzelnen Zeilen spezifiziert wird. Wenn der linke Knopf, im Dialog mit den Text "Beschreibung", betätigt wird, dann steht ganz unten in der Liste das Kriterium "Alle Konten". Wird das ausgewählt, dann ändern sich die Schaltflächen in der Zeile und es erscheint der Knopf "Konten Auswählen" und es öffnet sich ein kleiner Dialog.
Für diese unterschiedlichen Variationen gibt es dann im Handbuch diese tabellarische Auflistung.

Wenn ich diesen Abschnitt jetzt weiter nach hinten schiebe, dann würde das eher der Bearbeitungsreihenfolge entsprechen, wie der Suchen-Dialog genutzt wird.

Und ich könnte ja auch ein neues Bild mit z.B. zwei Suchkriterien erzeugen.

Reply:
Ich habe die Fußnote wieder in die Tabelle verschoben und den oberen Bereich eingekürzt. Meines Erachtens ist das so auch verständlich.